tests/sentry/api/test_paginator.py

```diff
@@ -631,6 +634,179 @@ def test_mix_date_and_not_date(self):
                 intermediaries=[rule_intermediary, rule_intermediary2],
             )
 
+    def test_only_issue_alert_rules(self):
+        project = self.project
+        Rule.objects.all().delete()
+        rule_ids = []
+
+        for i in range(1, 31):
+            rule = Rule.objects.create(id=i, label=f"rule{i}", project=project)
+            rule_ids.append(rule.id)
```

Comment:
Nit: This will be a bit slow, could we use `.bulk_create` to create these?

We could potentially use a fewer rules and a smaller `limit` as well.

(Same comments apply to the other new tests)

Reply:
Yeah I'll try it out, I mostly wanted to make it match reality as much as possible but we can probably go lower.